umbral-pre-wasm/src/lib.rs

```diff
@@ -393,7 +393,7 @@ impl KeyFrag {
     // So we have to use 4 functions instead of 1. Yikes.
 
     #[wasm_bindgen]
-    pub fn verify(&self, verifying_pk: &PublicKey) -> Result<VerifiedKeyFrag, JsValue> {
+    pub fn verify_with_only_public_key(&self, verifying_pk: &PublicKey) -> Result<VerifiedKeyFrag, JsValue> {
```

Comment:
You can use `#[wasm_bindgen(js_name = verifyWithOnlyPublicKey)]` tag before the method definition to make the name camel-case in JS.

Reply:
Maybe "with_only_verifying_key" for the name? I know it sounds like a tautology, but "public key" seems too vague.